src/sentry/tasks/post_process.py

```diff
@@ -815,20 +815,6 @@ def process_inbox_adds(job: PostProcessJob) -> None:
             not is_reprocessed and not has_reappeared
         ):  # If true, we added the .ONGOING reason already
             if is_new:
```

Comment:
are there any tests that need to be updated from this?

Reply:
nope - any tests that (incorrectly) assumed a new issue had the wrong substatus were updated in https://github.com/getsentry/sentry/pull/75471 where we explicitly set the substatus to NEW. the tests working as is is (hopefully) proof that these lines aren't needed.